jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -1075,12 +1075,16 @@ public static void load(String pathName) {
 	}
 /*[IF JAVA_SPEC_VERSION >= 15]*/
 	File fileName = new File(pathName);
+/*[IF PLATFORM-mz31 | PLATFORM-mz64]*/
+	ClassLoader.loadLibrary(getCallerClass(), fileName);
+/*[ELSE]*/
 	if (fileName.isAbsolute()) {
 		ClassLoader.loadLibrary(getCallerClass(), fileName);
 	} else {
 		/*[MSG "K0648", "Not an absolute path: {0}"]*/
 		throw new UnsatisfiedLinkError(Msg.getString("K0648", pathName));//$NON-NLS-1$
 	}
+/*[ENDIF] PLATFORM-mz31 | PLATFORM-mz64 */
 /*[ELSE]
 	ClassLoader.loadLibraryWithPath(pathName, ClassLoader.callerClassLoader(), null);
```

Comment:
Suggest inverting the `isAbsolute()` test which allows avoiding repetition:
```java
/*[IF !(PLATFORM-mz31 | PLATFORM-mz64)]*/
	if (!fileName.isAbsolute()) {
		/*[MSG "K0648", "Not an absolute path: {0}"]*/
		throw new UnsatisfiedLinkError(Msg.getString("K0648", pathName));//$NON-NLS-1$
	}
	ClassLoader.loadLibrary(getCallerClass(), fileName);
/*[ENDIF] !(PLATFORM-mz31 | PLATFORM-mz64) */
/*[ELSE] JAVA_SPEC_VERSION >= 15 */
```
Also note the comment added to `/*[ELSE]`.

Reply:
Updated.